styles.css

```diff
@@ -0,0 +1,88 @@
+.menu-logo div {
+  width: 25px;
+  height: 3px;
+  background-color: #6070ff;
+  margin: 4px 0;
+}
+
+.logo {
+  font-family: 'Poppins', "Times New Roman", Times, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.toolbar {
+  padding: 20px;
+  background-color: rgb(255, 255, 255);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  width: calc(100% - 45px);
+  height: 3rem;
+}
+
+.toolbar > a,
+p {
+  padding: 2rem;
+}
+
+body {
+  height: 100vh;
+  margin: 0;
+}
+
+.headline {
+  padding: 2rem;
+  height: 100vh;
+  background-image: url(images/header-background-mobile.png);
+  background-image: url(images/header-shapes\ mobile.png);
```

Comment:
- [ ] For some reason your design seems to be different from the Figma design

- Your design   
![image](https://github.com/agneta1991/Portfolio/assets/68823331/147d7b57-b3ba-439b-be92-0b1f80a0690a)

- Figma design
![image](https://github.com/agneta1991/Portfolio/assets/68823331/155c4e6a-8d0d-4baf-913d-44998cfb5672)

- [ ] I highly suggest you follow the Figma design to pixel perfection

Reply:
as i had issues with correct font family reference link, and after it has been updated, the color codes and design looks much closer to the template.